src/components/Layout/Menu.js

```diff
@@ -16,7 +16,6 @@ const getMenus = function (menuArray, siderFold) {
         <SubMenu key={linkTo.pathname}
           title={
             <span>
-              {item.icon ? <Icon type={item.icon} /> : ''}
```

Comment:
Personally, I prefer removing the icons, as most of them don't accurately fit the pages. 
Also, having no icons aligns better with modern web trendings.

Example of menu with icons
<img width="1920" height="970" alt="Screenshot 2025-07-29 at 4 12 38 PM (2)" src="https://github.com/user-attachments/assets/0f9e929e-2add-4e5b-bfe2-d979bacc47d7" />